pkg/remote/remote.go

```diff
@@ -104,9 +104,9 @@ RUN \
   /okteto/bin/okteto remote-run {{ .Command }} --log-output=json --server-name="${{ .InternalServerName }}" {{ .CommandFlags }}{{ if eq .Command "test" }} || true{{ end }}
 
 {{range $key, $artifact := .Artifacts }}
-RUN if [ -f /okteto/src/{{$artifact.Path}} ]; then \
```

Comment:
`-f` checks for a file, while `-e` checks for existance in general.

with `cp -r` we copy both files and dirs.